src/page_templates/account/LoginPage/LoginPage.strings.js

```diff
@@ -0,0 +1,13 @@
+//@flow
+const strings = {
```

Comment:
What is the advantage of having this in its own file?

Reply:
Mostly just to help reduce file length but also thought it might help at a later date if we decide to implement a more complex strings/translations system